### PR TITLE
Move e2e tests to us-east1

### PR DESCRIPTION
**Pull request by hongye-sun**
The quota is fulfilled.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/814)

### CODE REVIEW

**Comment by IronPan** [comment]
is it still needed given that tests are running fine in west1?

File: test/presubmit-tests-with-pipeline-deployment.sh [hunk 1]
```diff
@@ -71,7 +71,7 @@ echo "presubmit test starts"
 
 # activating the service account
 gcloud auth activate-service-account --key-file="${GOOGLE_APPLICATION_CREDENTIALS}"
-gcloud config set compute/zone us-west1-a
+gcloud config set compute/zone us-east1-a
```
Comment (IronPan) [inline test/presubmit-tests-with-pipeline-deployment.sh]: east1-a is not a valid region. please switch to east1-b/c

Comment (hongye-sun) [reply test/presubmit-tests-with-pipeline-deployment.sh]: done

**Comment by cjwagner** [comment]
Something is wrong with the `Travis CI - Pull Request` status context. It appears twice, one passing and the other pending. Tide is jammed on this PR as a result.
Holding this PR until it can be resolved to unblock Tide on this repo and stop Tide errors.
/hold

**Comment by hongye-sun** [comment]
@cjwagner, do you know how I can skip the dup pull request check for this PR?

**Comment by cjwagner** [comment]
Try pushing a dummy commit then removing it or closing and reopening the PR. Unfortunately I don't know much about Travis CI. It is strange to see two statuses with identical names though. I'm not sure how that could occur.

If the `override` plugin was enabled for this repo, repo admins would be able to force a status context to pass with a command like `/override Travis CI - Pull Request`

**Comment by cjwagner** [comment]
The "Travis CI - Pull Request" contexts aren't included in results from the commit status API: https://api.github.com/repos/kubeflow/pipelines/commits/1eb96089b29a7784ccf6e2edd9090c74b39551e9/status
It looks like they are actually GitHub Checks (see the tab next to the file view tab above). Tide doesn't currently support considering GitHub checks.

**Comment by hongye-sun** [comment]
Thanks. I will try close/reopen to see if it can refresh the status.

**Comment by hongye-sun** [comment]
Created https://github.com/kubeflow/pipelines/pull/847 to replace this one as this is blocked by the double travis build problem.

**Pull request closed without merging**